Header header
string robot_ID
bool sent_position
bool is_arrived
float32 position_x
float32 position_y
